lib/process.js

```diff
@@ -0,0 +1,27 @@
+exports.process = async ({ record, log, db, dynamo }) => {
+
+  const { maintainerId } = JSON.parse(record.body)
+
+  log.info('Starting process of maintainer payout %s', maintainerId)
+
+  // If another lambda has already picked up this transaction, it'll be locked on maintainer id
+  // preventing us from double paying maintainers.
+  // This will throw if it's locked
+  const lockInfo = await dynamo.lockOrg({ maintainerId })
+  log.info({ lockInfo })
+
+  const { payouts, ilpPaymentPointer } = await db.getPendingMaintainerPayouts({ maintainerId })
```

Comment:
probably should check if pointer is defined and unlock + exit early otherwise here